setup.py

```diff
@@ -19,6 +15,7 @@
     license='MIT',
     description='tools and library for Habitica restful API (http://habitica.com)',
     packages=['habitipy'],
+    python_requires='>=3.9',
```

Comment:
`importlib.resources.files` was only added in Python 3.9.
https://docs.python.org/3.13/library/importlib.resources.html#importlib.resources.files

However, 3.8 has already reached its EOL.